packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -666,7 +666,9 @@ export function TransferPanel() {
           receiveMessageTimestamp: null
         },
         parentChainId: parentChain.id,
-        childChainId: childChain.id
+        childChainId: childChain.id,
+        sourceChainId: isDepositMode ? parentChain.id : childChain.id,
```

Comment:
can we use `networks.sourceChain.id` instead? (same for destination)

Reply:
updated